### PR TITLE
added testdisk spk

### LINKED ISSUE #1407: [Request] TestDisk
seems like there is already a compiled binary for ds 211:
http://www.cgsecurity.org/wiki/TestDisk_Download

### PR DESCRIPTION
**Pull request by cytec**
fixes #1407

### CODE REVIEW

File: spk/testdisk/Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+SPK_REV = 1
+SPK_ICON = src/testdisk.png
+
+DEPENDS  = cross/$(SPK_NAME)
+
+RELOAD_UI = yes
+STARTABLE = no
+BETA = 1
+
+MAINTAINER = cytec
```
Comment (Dr-Bean) [inline spk/testdisk/Makefile]: SynoCommunity here :)

**Comment by Dr-Bean** [comment]
Just one comment ;) Other than that, with a squash we should be good imo.

**Comment by cytec** [comment]
@Dr-Bean fixed :)

**Comment by Dr-Bean** [comment]
Noticed that the digests were incorrect. Probably the developer updated the file?

**Pull request merged**